src/posh-git.psm1

```diff
@@ -87,7 +87,8 @@ if ($ForcePoshGitPrompt -or !$currentPromptDef -or ($currentPromptDef -eq $defau
         Write-VcsStatus
 
         # If stopped in the debugger, the prompt needs to indicate that in some fashion
-        $debugMode = (Test-Path Variable:/PSDebugContext) -or [runspace]::DefaultRunspace.Debugger.InBreakpoint
+        $hasInBreakpoint = [runspace]::DefaultRunspace.Debugger | Get-member -Name InBreakPoint -MemberType property
```

Comment:
Minor nit, can you upcase the `m` in `M`.   Also what version of PowerShell are you using?  Otherwise LGTM.